test/extended/util/framework.go

```diff
@@ -235,7 +236,7 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface,
 			}
 		}
 	}
-	return fmt.Errorf("the deploy did not finish within 3 minutes")
+	return fmt.Errorf("the deploy did not finish within %v minutes", maxWaitingTime)
```

Comment:
There's no reason to go through the overhead of creating a separate PR (or a separate commit, before you ask... :) ) for a minor fix like this.  

As it happens, @gabemontero already caught and fixed this anyway (which is why this PR is now needing a rebase).